public/_posts/_articles/yearn-finance/yvboost/es.md

```diff
@@ -0,0 +1,99 @@
+
+### Votación
+
+Una vez que los titulares de CRV votan para bloquear su CRV, cambiándolo a veCRV, pueden votar sobre varias propuestas del DAO y agrupar los cambios de parámetros que se propongan, o proponer sus propios cambios.
+
+Vale la pena señalar que el veCRV nativo no se puede transferir, y la única forma de obtenerlo es mediante el CRV bloqueado para votos.
+
+Puedes hacer staking de CRV en [Curve.fi](http://curve.fi/) y administrar activamente tus impulsos para fondos de liquidez por ti mismo, o puedes dejar que Yearn se encargue de hacer staking de CRV por ti con nuestras vaults dedicadas: yveCRV, y yvBOOST. Además, nuestros tokens de yVault son negociables y transferibles, a diferencia de las tokens de CRV en staking en [Curve.fi](http://curve.fi/).
+
+## La yVault yveCRV 
+
+![](image2.png?w=128&h=128)
+
+### Gana CRV con un mejor impulso
+
+Cuando un usuario deposita CRV en la vault, ese CRV se bloquea en la plataforma [Curve.fi](http://curve.fi/) como veCRV y se le devuelve al usuario una versión tokenizada de veCRV, yveCRV. Esta vault te otorga una parte continua de las tarifas comerciales de Curve. Cada semana, estas recompensas se pueden reclamar como 3Crv (token LP de 3pool en Curve).
+
+Puede hacerlo tu mismo directamente en [Curve.fi](http://curve.fi/), pero hay una muy buena razón por la que uno preferiría usar la yVault yveCRV : **¡más recompensas!**
+
+¿Cuánto más? ¡Tus recompensas a través de esta vault pueden ser más del doble! Yearn logra esto porque periódicamente dona el 10 % del CRV cultivado que gana a través de todas las estrategias basadas en [Curve.fi](http://curve.fi/) a esta vault yveCRV y permite que los depositantes de la vault yveCRV reclamen la parte de la tarifa del protocolo de Curve perteneciente a Yearn. Esto significa que otorgamos todas las recompensas de Yearn, que podríamos haber reclamado para el protocolo, a los depositantes de yveCRV, lo que aumenta tus recompensas semanales.
+
+Bloquear tus tokens CRV en la vault significa que delegas tu poder de voto de [Curve.fi](http://curve.fi/) a Yearn, quien ejecuta constantemente simulaciones para optimizar tus asignaciones de votos, lo que maximiza el rendimiento en todas las vaults, ¡lo que beneficia tus depósitos en otras vaults!
+
+**Ahora puedes reclamar tus recompensas y gastar ese dinero en mojitos mientras disfrutas de tu jubilación.** Sin embargo, otra opción que puedes desear es volver a agregar tus recompensas a la vault para aumentar tus ganancias e incluso puedes encontrar un botón "Retomar staking". ” para hacer precisamente eso. Podrías muy bien hacer esto manualmente, pero Yearn lo tiene cubierto aquí con...
```

Comment:
AS-IS: **Ahora puedes reclamar tus recompensas y gastar ese dinero en mojitos mientras disfrutas de tu jubilación.** Sin embargo, otra opción que puedes desear es volver a agregar tus recompensas a la vault para aumentar tus ganancias e incluso puedes encontrar un botón "Retomar staking". ” para hacer precisamente eso. Podrías muy bien hacer esto manualmente, pero Yearn lo tiene cubierto aquí con...

TO-BE: **Ahora puedes reclamar tus recompensas y gastar ese dinero en mojitos mientras disfrutas de tu jubilación.** Sin embargo, otra opción que puedes desear es volver a agregar tus recompensas a la vault para aumentar tus ganancias e incluso puedes encontrar un botón "Retomar staking" para hacer precisamente eso. Podrías muy bien hacer esto manualmente, pero Yearn lo tiene cubierto aquí con...

Reply:
done